homeassistant/components/wallbox/__init__.py

```diff
@@ -72,10 +71,10 @@ def _get_data(self):
                 CONF_MAX_CHARGING_CURRENT_KEY
             ]
 
-            filtered_data = {k: data[k] for k in CONF_SENSOR_TYPES if k in data}
+            filtered_data = {k: data[k] for k in SENSOR_TYPES if k in data}
```

Comment:
I don't think we need to filter the data here. The platforms will access the items that they need.